pkgs/development/tools/gofumpt/hello.go

```diff
@@ -0,0 +1,7 @@
+package main
```

Comment:
Does this ned to be checked in? Can it be inlined in the test?

Reply:
I have a strong hatred against multiline strings in shell, and to be honest I struggled a lot with the first, unformatted, heredoc... 

Now do the same but in a `test(1)` invocation - didn't feel like it. I've also never seen multiple line `test` code, it doesn't feel right :>

Reply:
If you want I can try to get it inlined - I do realize my reasons are purely selfish

Reply:
I don't feel that strongly about it and I'm not going to insist on changes, I just dislike checking in non-essential code in separate language specific files.

TBH I don't think the package should have this type of test in tree, I'd expect the test will break before the package does. I'd just do a `testVersion` smoke check or similar and leave it at that.

Reply:
I also think running tests from the upstream repo as part of our build should be enough.

Reply:
I figured it'd be less of a test but more of making the life of the reviewer easier. When I reviewed the update, I had to find some go repo (of course I couldn't think of one immediately, as it goes :)), clone it and then run it.

I don't think that the go formatting standards will be changing any time soon, especially with such simple code.

Reply:
If you're keeping the test as it is please please add yourself as maintainer to the package.